tests/test_app/TestApp/Model/Table/InvoicesTable.php

```diff
@@ -13,6 +13,7 @@ public function initialize(array $config)
             'contain' => [
                 'InvoiceItems.InvoiceItemProperties',
                 'InvoiceItems.InvoiceItemVariations',
+                'InvoiceTypes',
```

Comment:
If you want to revert the change in InvoiceTable.php, that is totally fine with me. The change in DuplicateBehaviour.php has some value in my opinion, since it prevents an SQL Error in some cases, that can be quite confusing if you are just a casual user of this plugin.